src/sentry/utils/snuba.py

```diff
@@ -138,6 +138,7 @@
     "<": "less",
     ">=": "greaterOrEquals",
     "<=": "lessOrEquals",
+    "IS NULL": "isNull",
```

Comment:
Why is this addition necessary?

Reply:
otherwise the test crashes in discover code before we get to resolve_tags. I am not sure why but the mapping seems to make sense to me